### DISCUSSION TITLE
Questions about re-direct notice on Performance Scans

### DISCUSSION THREAD

**Discussion by daphneyyt** [GoogleChrome/lighthouse#13690] [Q&A]
Hi, I have a question regarding a re-direct notice message I have been receiving when I conduct performance scans using Lighthouse.

This is the message I am receiving in the results: 

**The page may not be loading as expected because your test URL (https://www.bizfile.gov.sg/ngbbizfileinternet/faces/oracle/webcenter/portalapp/pages/TransactionMain.jspx?selectedETransId=C223&selEnityType=C) was redirected to https://www.bizfile.gov.sg/ngbbizfileinternet/faces/oracle/webcenter/portalapp/pages/TransactionMain.jspx?selectedETransId=C223&selEnityType=C&_afrLoop=22483905659136945&_afrWindowMode=0&Adf-Window-Id=d3uzcus2s&_afrFS=16&_afrMT=screen&_afrMFW=924&_afrMFH=754&_afrMFDW=1536&_afrMFDH=864&_afrMFC=8&_afrMFCI=0&_afrMFM=0&_afrMFR=120&_afrMFG=0&_afrMFS=0&_afrMFO=0. Try testing the second URL directly.**

**Questions:**
1. May we understand what is the cause of this issue?
2. Does this error message mean that Lighthouse was unable to check the website for performance? 
3. Can we proceed to still use this set of data to rectify any issues raised?

Appreciate your response!

Thank you
Daphne

**Comment by adamraine** [comment]
> May we understand what is the cause of this issue?

Lighthouse detected a redirect while navigating to the first URL. It looks like your page is redirecting but hiding the URL change or something.

> Does this error message mean that Lighthouse was unable to check the website for performance?

No, but it means that the redirect could be affecting your site performance.

> Can we proceed to still use this set of data to rectify any issues raised?

Yes but the redirect may be affecting those results.

**Comment by daphneyyt** [comment]
Hi @adamraine thank you very much for your response and clarifications. Appreciate it!
